TEStribute/ClientWrapper.py

```diff
@@ -1,6 +1,8 @@
 from drs_client_module import fetchDataObject
 from tes_client_module import fetchTasksInfo
 
+from distance import ReturnDistance
+
 tes_url = "http://193.166.24.111/ga4gh/tes/v1/"
```

Comment:
I know I've told you to hard-code these values, but I think now is actually a good time to start thinking about the actual use case. What is going to happen in practice is that the input parameters as defined in the proposal (basically the info in `tesResources`, a list of TES and DRS instances etc.) are passed to `TEStribute` by some wrapping code. What this means technically is that `TEStribute` needs to expose some function (or possibly class method, but I think a function is totally fine for now) that can be imported by the wrapping code and called with those parameters. And this is precisely what should be in this wrapping module: a function that takes the inputs and returns the outputs as defined in the proposal.

So I would suggest that you refactor `ClientWrapper.py` (and probably rename it to `__init__.py` so that one can import from it with something like `from TEStribute import rankServices`) so that all of the code (except imports and module doc string) go into a function (e.g., `rankServices()`), which then calls other functions as needed to e.g., get info from TES services, DRS services, calculate distances etc.. At the same time, you could create a mock script for calling the function in a directory `test`. This script would then basically mimic the wrapping code that in practice would eventually be something like `proTES`. All this script needs to do is call `rankServices()` with all required parameters and log the output. So where does it get the required parameters from? It should get them from some config `YAML` file, in which all of the hard-coded stuff goes: lists of TES and DRS IPs, task resource requirements etc. We can then start this script with different `YAML` config files to simulate different scenarios - those will be our tests.

So to summarize:
- rename `ClientWrapper.py` to `__init__.py`
- functionalize the code: implement a function signature and (for now) return `None`
- create a directory for `tests`
- create a script that imports the main `TEStribute` function, reads input params from a config file and calls the function with those params
- write a test (= config) file in `YAML` format

At this point I would consider the code stable, as you can call it and get some logging and, at this point, a return value of None. Then we can go from there and improve that step by step...